hub/powertoys/quick-accent.md

```diff
@@ -12,21 +12,19 @@ no-loc: [PowerToys, Windows, Quick Accent, Win]
 
 Quick Accent is an alternative way to type accented characters, useful when a keyboard doesn't support that specific accent with a quick key combo. This tool is based on [Damien Leroy's PowerAccent](https://github.com/damienleroy/PowerAccent).
 
-This utility is turned off by default. In order to use the Quick Accent utility, open PowerToys Settings, select the **Quick Accent** tab, and set the Enable toggle to "On".
+In order to use the Quick Accent utility, open PowerToys Settings, select the **Quick Accent** tab, and turn the **Enable** toggle on.
```

Comment:
Not sure about this one. I prefer "enable" to "turn on", but "enable the Enable toggle" sounds worse.

Reply:
Ok, I would leave the words "turn on" together like the style guide then, rather than leaving "on" as a dangling participle.

Reply:
Got a point there